packages/zmarkdown/__tests__/__snapshots__/new-html-suite.test.js.snap

```diff
@@ -1,5 +1,18 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`code highlight special cases does not highlight console 1`] = `
+"<div class=\\"hljs-code-div\\"><div class=\\"hljs-line-numbers\\"><span></span></div><pre><code class=\\"language-console\\">echo \\"Hello world\\"
+</code></pre></div>"
+`;
+
+exports[`code highlight special cases highlights latex as tex 1`] = `
+"<div class=\\"hljs-code-div\\"><div class=\\"hljs-line-numbers\\"><span></span><span></span><span></span><span></span></div><pre><code class=\\"hljs language-latex\\"><span class=\\"hljs-tag\\">\\\\<span class=\\"hljs-name\\">\\\\</span></span>usepackage{inputenc}[utf8]
+<span class=\\"hljs-tag\\">\\\\<span class=\\"hljs-name\\">\\\\</span></span>begin{document}
+<span class=\\"hljs-tag\\">\\\\<span class=\\"hljs-name\\">\\\\</span></span>texttt{code}
+<span class=\\"hljs-tag\\">\\\\<span class=\\"hljs-name\\">\\\\</span></span>end{document}
```

Comment:
From what I read here, only the `\` is highlighted, `end{document}` is not highlighted.

This seems wrong. TeX highlighting should at least detect `{}` and its content, and highlight this.

Reply:
If you have issues reading snapshots, you don't have to use snapshots.

Or before using a snapshot, you could check the snapshot would be. Try `.toBe(null)` instead of `.toMatchSnapshot()`, it will usually tell you that you expected the output to be `null` but got `…` instead.

Read carefully what `…` is, and if you are 100% sure it is correct, use a snapshot.